wzanalysis/wzanalysis.py

```diff
@@ -101,12 +111,11 @@ def init(self):
     for i, dataName in dataset.items(): 
       if dataName == name: self.sampleDataset = i
 
-    # Jet and lep pT
+    # Jet global pT cuts
     self.JetPtCut  = 20
-    self.LepPtCut  = 8
-    self.Lep0PtCut = 20
```

Comment:
Why this?

Reply:
They were not being used anywhere in the code